play/src/containers/FinalResultsContainer.tsx

```diff
@@ -21,14 +24,26 @@ export default function FinalResultsContainer({
   score,
   selectedAvatar,
   teamId,
-  leader,
 }: FinalResultsContainerProps) {
   const theme = useTheme();
   const isSmallDevice = useMediaQuery(theme.breakpoints.down('sm'));
   const [finalResultsState, setFinalResultsState] = useState(
     FinalResultsState.CONGRATS
   );
 
+  const isLeader = (inputTeams: ITeam[], inputTeamID: string) => {
+    inputTeams.sort((a, b) => b.score - a.score);
+    if (inputTeams.length <= 5) {
+      return true;
+    }
+    for (let idx = 0; idx < 5; idx += 1) {
```

Comment:
I like the idea here of constraining what we are searching to to just the first 5 entires (rather than mapping through the entire array for instance). We could still make use of array methods here with something like this though:

`    if (inputTeams.slice(0, 5).find((team) => team.id === inputTeamID))
      return true;`
      
Feel like this might be more compact than the for loop

Reply:
Yes, that's so much better